### PR TITLE
PIPRES-180: Order state duplicate fix

### PR DESCRIPTION
**Pull request by mandan2**
Improved order state install process: 
- Moved logic to separate service
- Used single method for any order state install.
- Created test to check if these states exist.

### CODE REVIEW

File: src/DTO/OrderStateInstallerData.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Mollie\DTO;
+
+class OrderStateInstallerData
```
Comment (tomjas1997) [inline src/DTO/OrderStateInstallerData.php:5]: OrderStateData IMO, it's not installer data.

File: src/Exception/CouldNotInstallModule.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Mollie\Exception;
+
+class CouldNotInstallModule extends MollieException
+{
+    const FAILED_TO_INSTALL_ORDER_STATE = 1;
```
Comment (tomjas1997) [inline src/Exception/CouldNotInstallModule.php:7]: Maybe add ExceptionCode?

File: src/DTO/OrderStateInstallerData.php (hunk 1 above)
Comment (mandan2) [reply src/DTO/OrderStateInstallerData.php]: renamed

File: src/Exception/CouldNotInstallModule.php (hunk 2 above)
Comment (mandan2) [reply src/Exception/CouldNotInstallModule.php]: Ok, in the future exceptions will be refactored so hopefully I will be able to separate main logic and infrastructure.

**Review by tomjas1997 — APPROVED** [review]

**Pull request merged**